doc/specs/#1235 - Azure cloud shell connector/spec.md

```diff
@@ -53,7 +53,7 @@ This feature will not impact reliability of Windows Terminal.
 
 ### Compatibility
 
-With the implementation being mostly decoupled from the Windows Terminal app itself, no existing code/behaviours should break due to this feature. 
+With the implementation being mostly decoupled from the Windows Terminal app itself, no existing code/behaviors should break due to this feature. 

```

Comment:
This corpus is almost entirely en-US, and the en-US spelling is dominant. That said, I can split/drop this if requested.

Reply:
As a native en-UK speaker, I do make an effort to use en-US spelling on this project, but I'm sure I slip up from time to time. It's not my call to make, but I'd personally have no problem with having my spelling corrected (I know I'm to blame for at least one of the "behaviours" picked up in this PR).

Reply:
> Generated by [jsoref/spelling](https://github.com/jsoref/spelling) `f`; to maintain your repo, please consider `fchurn`

@jsoref, is there a way we can run this as a part of our CI and make it fail on a typo like the clang-tidy style rules do?

Reply:
We would prefer to do the `en-US` just to be consistent, generally speaking.

We've had our share of `en-UK` from @bitcrazed dropped into assorted things, though. He very insistently says that Americans were given a perfectly good language that we decided to not use. 

I don't mind the mix of them. But if you want to enforce one, lean toward `en-US`.